docs/sphinx/source/whatsnew/v0.6.1.rst

```diff
@@ -73,6 +74,8 @@ Bug fixes
 * Fix documentation errors when using IPython >= 7.0.
 * Fix error in :func:`pvlib.modelchain.ModelChain.infer_spectral_model` (:issue:`619`)
 * Fix error in ``pvlib.spa`` when using Python 3.7 on some platforms.
+* Fix error in :func:`pvlib.irradiance._delta_kt_prime_dirint` (:issue:`637`). The error affects
+  the first and last values of DNI calculated by the function :func:`pvlib.irradiance.dirint`
```

Comment:
I'm concerned and confused to see this and the associated changes in `irradiance.py` show up in this PR. Can you merge `master` into your branch and see if it cleans up the PR?

Reply:
I'm confused too especially since I haven't made changes to irradiance.py at all. 

When I do a diff on my branch with upstream/master this isn't highlighted.